FirebaseStorage/Sources/FIRStorageUtils.h

```diff
@@ -108,4 +108,14 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+@interface NSURL (FIRStorageNSURLHelpers)
+
+/**
+ * Returns a boolean indicating that the NSURL instance represents a file directory
+ * @return True if the NSURL object represents a file directory, and false if otherwise
+ */
+- (BOOL)frs_hasDirectoryPath;
```

Comment:
Nit: The Firebase prefix for all Firebase API classes is `fir`. Should we use the same prefix here?

Reply:
Good point, I can fix that. FWIW, I prefixed it with `frs_` because I noticed the other categories in `FIRStorageUtils.h` had [methods prefixed with `frs_`](https://github.com/firebase/firebase-ios-sdk/blob/8d1e2464255365a88619225740ffedcf01195428/FirebaseStorage/Sources/FIRStorageUtils.h#L107). Just curious, was there a reason some of the other category methods in `FIRStorageUtils.h` were prefixed with `frs_`?